rebar.config

```diff
@@ -4,7 +4,7 @@
 {eunit_opts, [verbose]}.
 {erl_opts, [warnings_as_errors, debug_info, nowarn_deprecated_type]}.
 {deps, [
-		{riak_pb, "2.1.1.0", {git, "git://github.com/basho/riak_pb", {tag, "2.1.1.0"}}}
+		{riak_pb, "2.1.1.0", {git, "git://github.com/basho/riak_pb", {tag, "2.1.1.0"}}} 
```

Comment:
We'll want this to pull from `end-to-end/timeseries`, not `2.1.1.0`, to get the latest PB changes.